docs/developer/checkout.mdx

```diff
@@ -274,23 +275,30 @@ As a result, we get the payment object:
 
 ## Completing the checkout
 
-This operation requires only the checkout ID as an input. Its purpose is to ensure this checkout is correct. To do this, it verifies if:
+The purpose of the operation is to ensure this checkout is correct. To do this, it verifies if:
 
 1. The required addresses are valid.
 
 2. All selected products are in stock (while making the purchase, another user could already buy the last available item).
 
 3. The payment creation succeeded.
 
+4. No additional actions are required.
```

Comment:
What actions? I would add a simple example - "No additional actions are required (e.g. 3DS confirmation)".

Reply:
Some of our community members can ask about it. So adding some examples would be a good idea.